build/ical.js

```diff
@@ -6668,6 +6681,416 @@ ICAL.Event = (function() {
   return Event;
 
 }());
+
+ICAL.Todo = (function() {
```

Comment:
I assume a lot of this is copy/paste from ICAL.Event? If so, it would probably make sense to create a BaseItem class. I don't think the BaseItem class needs to be exported on the ICAL object unless its needed to keep it in a separate file.